CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 ### Changes (non-breaking)
 ### Bug fixes
 
+## 4.1.1 - [release](https://github.com/LuccaSA/lucca-ui/releases/tag/4.1.1)
```

Comment:
As we will do one merge, please put all your release notes in the "in dev" section. I will do the job of versionning